### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by calebk1**
Starting work for the split.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -273,117 +154,6 @@ module.exports = {
...
-                            },
-                            {
-                                title: "Metrics",
-                                path: "documentation/lifecycle-for-edge-network/metrics"
-                            }
-                        ]
-                    },
-                    {
-                        title: "Adobe Experience Platform Assurance",
-                        path: "/documentation/platform-assurance-sdk",
```
Comment (emdobrin) [inline gatsby-config.js:362]: we should keep the relevant pages for assurance sdk

File: gatsby-config.js [hunk 2]
```diff
@@ -581,40 +351,6 @@ module.exports = {
                             }
```
Comment (emdobrin) [inline gatsby-config.js:351]: For all the extensions let's remove event reference from acp repo

File: gatsby-config.js [hunk 3]
```diff
@@ -826,101 +414,13 @@ module.exports = {
...
-                            {
-                                title: "Push notifications",
-                                path: "documentation/troubleshooting-guides/push-notifications"
-                            }
-                        ]
-                    },
                     {
                         title: "Frequently asked questions",
                         path: "documentation/faq"
                     },
```
Comment (emdobrin) [inline gatsby-config.js:420]: let's keep this in aep repo

File: gatsby-config.js [hunk 4]
```diff
@@ -826,101 +414,13 @@ module.exports = {
...
                     {
                         title: "Migrate to Swift",
                         path: "documentation/migrate-to-swift"
-                    },
-                    {
-                        title: "Privacy and GDPR",
-                        path: "documentation/privacy-and-gdpr"
-                    },
```
Comment (emdobrin) [inline gatsby-config.js:857]: we should keep the gdpr file too -> Using Adobe Experience Cloud Solution extensions section

File: gatsby-config.js [hunk 5]
```diff
@@ -646,154 +382,6 @@ module.exports = {
                     }
```
Comment (emdobrin) [inline gatsby-config.js:382]: For Target we can remove event reference. And we can keep Deprecated APIs here and maybe remove from AEP repo, @swarna04 what do you think?

File: gatsby-config.js [hunk 6]
```diff
@@ -58,55 +58,6 @@ module.exports = {
...
                 ]
             },
-            {
-                title: "Version 4 Mobile SDKs end-of-support FAQ",
-                path: "/documentation/v4-end-of-life-faq"
-            },
-            {
-                title: "Version 4 API changelog",
-                path: "/documentation/api-changelog"
-            },
```
Comment (emdobrin) [inline gatsby-config.js:68]: I am thinking we can keep these two here, @swarna04 thoughts on this?

File: src/pages/documentation/experience-cloud-extensions.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+## Extensions
+
+- [Adobe Analytics](./adobe-analytics/index.md)
+- [Adobe Analytics - Mobile Services](./adobe-analytics-mobile-services/index.md)
+- [Adobe Media Analytics](./adobe-media-analytics/index.md)
+- [Adobe Audience Manager](./adobe-audience-manager/index.md)
+- [Adobe Campaign Classic](./adobe-campaign-classic/index.md)
+- [Adobe Campaign Standard](./adobe-campaign-standard/index.md)
+- [Adobe Journey Optimizer](./adobe-journey-optimizer/index.md)
+- [Adobe Journey Optimizer Decisioning](./adobe-journey-optimizer-decisioning/index.md)
```
Comment (swarna04) [inline src/pages/documentation/experience-cloud-extensions.md:14]: AJO & AJO Decisioning can be removed as they have only Swift SDKs.

File: gatsby-config.js (hunk 6 above)
Comment (swarna04) [reply gatsby-config.js:68]: Until we get rid of them, let's have it in both places. Sounds good?

File: gatsby-config.js (hunk 3 above)
Comment (swarna04) [reply gatsby-config.js:420]: And curate it for AEP specific questions, later maybe.

File: gatsby-config.js [hunk 8]
```diff
@@ -374,33 +196,14 @@ module.exports = {
                             },
                             {
                                 title: "Resolving common issues",
-                                path: "/documentation/platform-assurance-sdk/error-logs"
+                                path: "/documentation/platform-assurance-sdk/common-issues/"
```
Comment (emdobrin) [inline gatsby-config.js:199]: i'd say keep this for aep repo only

File: gatsby-config.js [hunk 9]
```diff
@@ -274,86 +163,19 @@ module.exports = {
                         ]
```
Comment (emdobrin) [inline gatsby-config.js:163]: L152 remove Event reference for mobile core

File: gatsby-config.js [hunk 10]
```diff
@@ -517,10 +316,6 @@ module.exports = {
                                 title: "API reference",
```
Comment (emdobrin) [inline gatsby-config.js:316]: L257 For analytics above - let's remove product variable and event reference

**Comment by calebk1** [comment]
I'm merging this in as is so @hbhandal80 will be able to work on this!

**Pull request merged**